client/state/sites/media-storage/test/selectors.js

```diff
@@ -57,4 +58,44 @@ describe( 'selectors', () => {
 			expect( isRequestingMediaStorage( state, 'not-defined' ) ).to.eql( false );
 		} );
 	} );
+	describe( '#isOverMediaLimit()', () => {
+		it( 'should return true if a site is over storage limits', () => {
+			const state = deepFreeze( {
+				sites: {
+					mediaStorage: {
+						items: {
+							2916284: {
+								max_storage_bytes: 3221225472,
+								storage_used_bytes: 3221225900
+							},
+							77203074: {
+								max_storage_bytes: 3221225472,
+								storage_used_bytes: 323506
+							}
+						}
+					}
+				}
+			} );
+
+			expect( isOverMediaLimit( state, 2916284 ) ).to.eql( true );
```

Comment:
A common argument for why `sinon` is a bit of a pain to work with, but there are subtle differences between `equal` and `eql`, particularly that the latter performs a deep comparison. When strict comparison is all you need (i.e. non-complex objects), probably best to use `equal`.